ffmpeg/transcoder.c

```diff
@@ -271,9 +274,21 @@ int transcode(struct transcode_thread *h,
   }
 
   // flush outputs
-  for (i = 0; i < nb_outputs; i++) {
-    ret = flush_outputs(ictx, &outputs[i]);
+  for (i = 0; i < nb_outputs; i++) {  
+#ifdef USE_LVPDNN_  
+    if(outputs[i].is_dnn_profile == 0 ){
+      ret = flush_outputs(ictx, &outputs[i]);
+      if (ret < 0) LPMS_ERR(transcode_cleanup, "Unable to fully flush outputs")
+    } 
+    else if(outputs[i].is_dnn_profile && outputs[i].res->frames > 0){      
+       for (int j = 0; j < MAX_CLASSIFY_SIZE; j++){
+         outputs[i].res->probs[j] =  outputs[i].res->probs[j] / outputs[i].res->frames;         
```

Comment:
Just to make sure I understand this correctly:

- There will be `j` probability values where j = the # class IDs for the lvpdnn filter
- The jth probability is the probability score for the jth class ID
- We divide the jth probability by the total frames sampled

Is that right?

Reply:
Bump